pkg/rbd/rbd_util.go

```diff
@@ -125,12 +126,20 @@ func createImage(ctx context.Context, pOpts *rbdVolume, volSz int64, cr *util.Cr
 		klog.V(4).Infof(util.Log(ctx, "rbd: create %s size %s format %s (features: %s) using mon %s, pool %s"),
```

Comment:
(nit) Most often our image format is 2, as a matter of fact there was a discussion not to support v1 formats anyway with @dillaman . Considering this, the log message will miss out the data pool argument in most cases, as it gets appended to the message in the else condition. Do we want this corrected? (I am fine either way as the RPC payload message for CreateVolume will contain the arguments).

Reply:
@ShyamsundarR in a perfect world (as a separate PR), the `imageFormat` option should be removed since V1 images are not supported and cannot be created (and V1 doesn't support data pools). Otherwise agree that the log message should move.